tests/test.js

```diff
@@ -51,58 +51,88 @@ describe("Embroider build", function () {
           }
         }
       });
+
+      if (scenario === "default") {
+        it("adds integrity attribute to link elements for alternate stylesheets", function () {
+          for (const el of indexHtml.querySelectorAll("link[rel='alternate stylesheet']")) {
```

Comment:
Do you mind having the value within double quotes? I feel that is more the established standard and eases readability.

```suggestion
          for (const el of indexHtml.querySelectorAll('link[rel="alternate stylesheet"]')) {
```

It's also used by all examples within MDN: https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#values Even if quotes are not strictly required because value does not contain a space or special character.

Reply:
Ah looks like the ones at the top of the loop didn't have the double quotes so I copied that, but I see it was different elsewhere in the file - I updated all of them to be consistent.